nursery/communicate-with-kernel-module-via-netlink-socket-on-linux.yml

```diff
@@ -0,0 +1,13 @@
+rule:
+  meta:
+    name: communicate with kernel module via Netlink socket on Linux
+    namespace: host-interaction/kernel
```

Comment:
not sure the namespace is great, but cannot think of anything else right now

Reply:
Agreed -  I had the same thoughts when creating the rule. I'll leave it as-is for now.